profiles/Sadeedpv.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/96517901/181920820-ecc4d039-6a4a-436c-83d5-a1d12707ce79.png",
+  "issueId": 1270,
+  "name": "Sadeed pv",
+  "username": "Sadeedpv"
+}
```

Comment:
Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient